docs/guides/options.md

```diff
@@ -1,138 +1,351 @@
+
+### `fluid`
+> Type: `boolean`
+
+When `true`, the Video.js player will have a fluid size. In other words, it will scale to fit its container.
+
+Also, if the `<video>` element has the `"vjs-fluid"`, this option is automatically set to `true`.
+
+### `inactivityTimeout`
+> Type: `number`
+
+Video.js indicates that the user is interacting with the player by way of the `"vjs-user-active"` and `"vjs-user-inactive"` classes and the `"useractive"` event.
+
+The `inactivityTimeout` determines how many milliseconds of inactivity is required before declaring the user inactive. A value of `0` indicates that there is no `inactivityTimeout` and the user will never be considered inactive.
+
+### `language`
+> Type: `string`, Default: browser default or `'en'`
+
+A [language code][lang-codes] matching one of the available languages in the player. This sets the initial language for a player, but it can always be changed.
+
+Learn more about [languages in Video.js](languages.md).
+
+### `languages`
+> Type: `Object`
+
+Customize which languages are available in a player. The keys of this object will be [language codes][lang-codes] and the values will be objects with English keys and translated values.
+
+Learn more about [languages in Video.js](languages.md).
+
+> **Note**: Generally, this option is not needed and it would be better to pass your custom languages to `videojs.addLanguage()`, so they are available in all players!
+
+### `nativeControlsForTouch`
+> Type: `boolean`
+
+Explicitly set a default value for [the associated tech option](#nativecontrolsfortouch).
+
+### `notSupportedMessage`
+> Type: `string`
+
+Allows overriding the default message that is displayed when Video.js cannot play back a media source.
+
+### `plugins`
+> Type: `Object`
+
+This supports having plugins be initialized automatically with custom options when the player is initialized - rather than requiring you to initialize them manually.
+
+```js
+videojs('my-player', {
+  plugins: {
+    foo: {bar: true},
+    boo: {baz: false}
+  }
+});
 ```
 
-Finally, if you're not using the data-setup attribute to trigger the player setup, you can pass in an object with the player options as the second argument in the javascript setup function.
+The above is roughly equivalent to:
 
 ```js
-videojs("example_video_1", { "controls": true, "autoplay": false, "preload": "auto" });
+var player = videojs('my-player');
+
+player.foo({bar: true});
+player.boo({baz: false});
 ```
 
+Although, since the `plugins` option is an object, the order of initialization is not guaranteed!
 
-Individual Options
-------------------
+See [the plugins guide](plugins.md) for more information on Video.js plugins.
 
-> ### Note on Video Tag Attributes ###
-> With HTML5 video tag attributes that can only be true or false (boolean), you simply include the attribute (no equals sign) to turn it on, or exclude it to turn it off. For example, to turn controls on:
+### `sourceOrder`
+> Type: `Array`
```

Comment:
Also, should we mention now that in 6.0 it will "default" to true, whatever that means in the world of middleware.